hero-dev/install.yaml

```diff
@@ -1,6 +1,6 @@
 # Simulation
 - type: target
-  name: ros-tmc-desktop-full
+  name: ros-tmc_desktop_full
```

Comment:
@MatthijsBurgh Why make this change in this PR?

Reply:
I don't know why it is in this PR, but the changes are correct. ROS packages shouldn't contain dashes.

Reply:
Changes are definitely correct. I am just wondering, how this did not give any errors till now.

Reply:
Because the target itself has been renamed too